### PR TITLE
Ddce-5554 Play Upgrade

### PR DESCRIPTION
**Pull request by WozCapGem**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -24,6 +24,7 @@ play.http.errorHandler = "handlers.ErrorHandler"
 # ~~~~
 # Additional play modules can be added here
 play.modules.enabled += "uk.gov.hmrc.play.bootstrap.HttpClientModule"
```
Comment (adageorg) [inline conf/application.conf:26]: Do we still need the original `HttpClientModule` enabled if the V2 module has been introduced with the bootstrap upgrade?

**Review by adageorg — COMMENTED** [review]
Looks alright on the whole, but I have just a few suggestions/queries.

File: app/controllers/register/leadtrustee/individual/NameController.scala [hunk 2]
```diff
@@ -84,7 +83,7 @@ class NameController @Inject()(
               }
             case Failure(_) =>
               logger.error("[NameController][onSubmit] Error while storing user answers")
-              Future.successful(InternalServerError(errorPageView()))
+              errorHandler.onServerError(request, new Exception("Error while storing user answers."))
```
[the remark this answers is not in the document]
Comment (WozCapGem) [reply app/controllers/register/leadtrustee/individual/NameController.scala]: so it change was made so the service would compile but after the tech session today I learn how to deal with the code error that was there before

**Review by adageorg — APPROVED** [review]
OK, looks good.
For what it's worth, also as mentioned on the tech session yesterday, if you're using a Future to return another Future you can do a plain map so:
`errorHandler.internalServerErrorTemplate.flatMap(html => Future.successful(InternalServerError(html)))` 
...can be simplified to:
`errorHandler.internalServerErrorTemplate.map(html => InternalServerError(html))`

Although what you've got works just as well, so happy to approve.

**Review by adageorg — APPROVED** [review]

**Pull request merged**